project_liberation/main_page/templates/main_page/footer.haml

```diff
@@ -1,2 +1,51 @@
-%br
-  footer
\ No newline at end of file
+.footer.footer_color
+    .container
+        .row
+            .col-lg-3.offset-lg-0.col-md-3.offset-md-2.col-sm-4.offset-sm-4.col-6.offset-3
+                %section.widget.footer_logo
+                    %img{:src => "/static/main_page/images/footer/code_poets_logo_footer_white.png", :width => "208px", :height => "68px"}
+            .col-lg-3.offset-lg-0.col-md-3.offset-md-2.col-sm-4.offset-sm-4.col-6.offset-3
+                %section.widget.footer_address
+                    %h3.widgettitle Code Poets sp. z o.o.
+                    .textwidget.custom-html-widget
+                        ul. Józefa Piłsudskiego 49-57
+                        %br
+                        50-032 Wrocław
+                        %br
+                        Poland
+                        %br
+                        %br
+            .col-lg-3.offset-lg-0.col-md-3.offset-md-2.col-sm-4.offset-sm-4.col-6.offset-3
+                %section.widget.footer_contact
+                    %h3.widgettitle Contact Us
+                    .textwidget.custom-html-widget
+                        .anti-bot-style ti.steopedoc@tcatnoc
+                %section.widget.footer_contact-2
+                    .textwidget
+                        .social-media-container
+                            %a{:href => "https://www.facebook.com/codepoets.it/"}
+                                %img{:src => "/static/main_page/images/footer/social_icons/facebook_icon.png", :alt => "Facebook", :target => "_blank", :width => "25px", :height =>"25px"}
```

Comment:
This should not be directly path to file. Please use `static` template tag from django.

Reply:
Done